docs/configuration.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[configuration]]
+== Configuration
+
+To adapt the Elastic APM agent to your needs, you can configure it using environment variables.
+
+
+[[config-reporter]]
+=== Reporter configuration options
+[float]
+[[config-server-urls]]
+==== `server_urls`
+
+[options="header"]
+|============
+| Environment variable name
+| `ELASTIC_APM_SERVER_URLS`
+|============
+
+[options="header"]
+|============
+| Default                 | Type
+| `http://localhost:8200` | List
+|============
+
+The URLs for your APM Servers. The URLs must be fully qualified, including protocol (`http` or `https`) and port. To add multiple servers separate them with comma (`,`).
+
+
+NOTE: Providing multiple URLs only works if intake API v2 is enabled.
```

Comment:
I'm wondering if it might be better to mention 6.5 instead of v2:
```suggestion
NOTE: Providing multiple URLs only works with APM Server v6.5+.
```
Actually, is your agent even going to be compatible with less than 6.5? Might make more sense to put this elsewhere

Reply:
Currently it works with less than 6.5 - or to be precise, it works with intake v1. I'd keep it with your suggestion here. If we kill v1 support in .NET (atm I don't plan to kill it) then this sentence is redundant, but still not incorrect, but if we keep v1 support then it's helpful.